.github/workflows/ai-revision.yaml

```diff
@@ -0,0 +1,54 @@
+name: ai-revision
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Branch to revise'
+        required: true
+        type: string
+        default: 'main'
+      file_names:
+        description: 'File names to revise'
+        required: false
+        type: string
+        default: ''
+      model:
+        description: 'Language model'
+        required: true
+        type: string
+        default: 'text-davinci-003'
+      branch_name:
+        description: 'Output branch'
+        required: true
+        type: string
+        default: 'ai-revision-davinci'
+
+jobs:
+  chatgpt-revise:
+    name: GPT Revise
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+        with: 
+          ref: ${{ inputs.branch }} 
+      - name: Install Manubot AI Editor
+        run: bash ci/install-manuscript-editor.sh
+      - name: Revise manuscript
+        env:
+          OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
+          AI_EDITOR_LANGUAGE_MODEL: ${{ inputs.model }}
+          AI_EDITOR_FILENAMES_TO_REVISE: ${{ inputs.file_names }}
+          # More variables can be specified to control the behavior of the model:
+          #  https://github.com/greenelab/manubot-ai-editor/blob/main/libs/manubot/ai_editor/env_vars.py
```

Comment:
This URL currently gives a 404. We could change it to https://github.com/greenelab/manubot-ai-editor/blob/main/libs/manubot_ai_editor/env_vars.py now or wait until the repository is possibly transferred to the manubot org.

A related comment (somewhat out of scope for this pull request) is that I'm not sure what possible values `SECTIONS_MAPPING` can take.

Reply:
I changed the URL to the newly transfered [manubot/manubot-ai-editor](https://github.com/manubot/manubot-ai-editor) repo.

Reply:
Regarding `SECTIONS_MAPPING`, there is an example in the linked file. For instance: `{"01.intro.md": "introduction", ...}`

Reply:
Moving the out of scope part of this discussion to a new issue